### DISCUSSION TITLE
ERROR: The virtual machine or container JSON document is invalid

### DISCUSSION THREAD

**Discussion by superpi15** [microsoft/WSL#8269] [Q&A]
Hi friends, 
I'm using Win 11 pro. My WSL2 installs Ubuntu-20.04. 
It works fine for several months. 
Everyday I simply coding and use gcc compiler did not touch system related things. 
However, today when I boot WSL, it suddenly says: 

> The virtual machine or container JSON document is invalid.
> 
> [process exited with code 4294967295 (0xffffffff)]

I don't have any clue what happened here. 
I haven't done Windows update for several weeks, so update is less likely the reason. 
Any idea? 
Thanks.

**Comment by superpi15** [comment]
Update:
when I tried to reinstall WSL, I encounter this error:
> WslRegisterDistribution failed with error: 0x8037010d
> Error: 0x8037010d The virtual machine or container JSON document is invalid.

**Reply by sahith1311** [reply]
same problem any solution

**Comment by superpi15** [comment]
Update: 
I tried several things but none of them work. 
The last attempt, which is a random try,  luckily works. 
I switch from Insider dev channel to Beta channel and let update and restoration or changes happen. 
It seems to do something that beyond user's ability and wsl is fixed. 
Now wsl finally boot successfully. 
But some of my previous setting related to PulseAudio does not work anymore. 
So I use a clean installation of Ubuntu.

**Reply by alex-guo-github** [reply]
Did you try to put WSL into the bridge network? I used .wslconfig to do so and received the same error. I suspect that it might be an arbitrary limit. I have windows 10 22H2, and WSL 1.0.

**Reply by Quadre** [reply]
I got the same problem, and resolution was to remove Bridge network config from `.wslconfig` file.
Windows 10 22h2, WSL 2.0
